src/domain/payments/utils.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  MAX_CENTS,
+  MAX_SATS,
+  paymentAmountFromNumber,
+  WalletCurrency,
+  BtcAmountTooLargeError,
+  UsdAmountTooLargeError,
+} from "@domain/shared"
+
+import { InvalidBtcPaymentAmountError, InvalidUsdPaymentAmountError } from "./errors"
+
+export const checkedToBtcPaymentAmount = (
```

Comment:
`utils` is generally speaking a very bad name as it does not help in communicating structure or domain concepts.
These functions should either be grouped according to their function (eg. `validation`) or - even better - colocated with their related domain concept (`PaymentAmount`).